packages/app/src/Constants.tsx

```diff
@@ -1,6 +1,7 @@
-export const APP_NAME = 'zkSNARK Sudoku';
+export const APP_NAME = 'zkSNARK Sudoku x zkVerify';
+export const ORIGINAL_APP = 'zkSNARK Sudoku'
 export const APP_DESCRIPTION =
-  'This is zkSNARK Sudoku play and verify application.';
+  'This is zkSNARK Sudoku: play and verify the proof on zkVerify.';
 
 export const GMAIL = 'mailto:dany.armstrong90@gmail.com';
 export const LINKEDIN = 'http://www.linkedin.com/in/daniel-armstrong90';
```

Comment:
`export const GITHUB_PROJECT = 'https://github.com/HorizenLabs/zksnark-sudoku';`

This is used on the GitHub logo on the header

Reply:
I left that on purpose along with the original devs details wasn't sure how much I should change or keep the reference to original project.